app.css

```diff
@@ -37,12 +48,12 @@ a {
     background-size: cover;
 }
 
+#bitmoji-headshot {
+    max-width: 60vw;
+}
+
 .footer {
-    position: absolute;
-    padding-top: 40px;
-    padding-bottom: 40px;
     width: 100%;
     text-align: center;
-    background-color: lavender;
-    background-size: cover;
-}
\ No newline at end of file
+    padding-bottom: 20px;
```

Comment:
ensures that there's some breathing room between the footer content and the bottom of the page